src/parsing/parser.ts

```diff
@@ -188,6 +204,93 @@ export function parseRuleSyntax(
   };
 }
 
+export function parseMappedTrackerSyntax(
+  syntax: MappedTrackerJSON
+): MappedTrackerDefinition {
+  let keyType = syntax.keyType.trim();
+  let valueType = syntax.valueType.trim();
+  var trackerInitialKeys: any[] = [];
+  var trackerInitialValues: any[] = [];
+  for (var pair of syntax.initialvalues) {
+    if (keyType == "uint256") {
```

Comment:
Could lines 215-240 and 242-271 be refactored into a single function formatter function which could then be used with reduce for the assignments on line 212 and 213
```
[const trackerInitialKeys, trackerInitialValues] = syntax.initialValues.reduce(formatValues, [[],[]])